<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualVers</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>360</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/images/img/title.png</pixmap>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>C:\user/10923\windows\destop</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="BDiagPattern">
      <color alpha="255">
       <red>134</red>
       <green>152</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C:\Users\86186\Desktop\Qt</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="background">
     <brush brushstyle="BDiagPattern">
      <color alpha="255">
       <red>134</red>
       <green>152</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#openButton{
	padding:3px;
	background-color: rgb(226, 255, 253);
	color: rgb(95, 95, 95); 
	border-radius: 15px;  
	border: 2px groove gray;
	border-style: outset;
	
	font: 75 15pt &quot;微软雅黑&quot;;
	
}
#openButton:hover{
	background-color:rgb(139, 255, 242);
	color: rgb(0, 0, 0);
	padding:3px;   
	border-radius: 15px;  
	border: 2px groove gray;
	border-style: outset;
	font: 75 15pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#newButton{
	padding:3px;
	background-color: rgb(226, 255, 253);
	color: rgb(95, 95, 95); 
	border-radius: 15px;  
	border: 2px groove gray;
	border-style: outset;
	
	font: 75 15pt &quot;微软雅黑&quot;;
	
}
#newButton:hover{
	background-color:rgb(139, 255, 242);
	color: rgb(0, 0, 0);
	padding:3px;   
	border-radius: 15px;  
	border: 2px groove gray;
	border-style: outset;
	font: 75 15pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>最近项目</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
